paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerModule.kt

```diff
@@ -58,6 +60,16 @@ internal object FlowControllerModule {
         return confirmationHandlerFactory.create(viewModel.viewModelScope)
     }
 
+    @Provides
+    @Singleton
+    fun providesExpressCheckoutContent(
```

Comment:
Looks like there was a rename in the middle of this. Can we scrub this PR of "express checkout"?